_src/doxygen/doxygen.css

```diff
@@ -193,8 +203,9 @@ div.fragment {
 
 div.fragment div.line {
 	font-family: monospace, fixed;
-        font-size: 85%;
-        color: #ddd;
+	font-size: 85%;
+	font-weight: 900;
+	color: #616161;
```

Comment:
This seems a little light on a background of #dedede, maybe we should make this darker?  Example:
http://mlpack2.ratml.org/doc/mlpack-3.1.1/doxygen/timer.html

Also personally I think the `font-weight` makes it too thick, just my opinion.  Although I guess I can see it either way.  So that's certainly not a strong opinion.

Reply:
I'm not sure what you refer to, do you have an example to see where the problem is? (tracking doxygen css is painful).

Reply:
Oh, sorry, I just meant the code snippets there.  For instance the first one on the page:

```
void Timer::Start(const char* name);
void Timer::Stop(const char* name);
timeval Timer::Get(const char* name);
```

I think the gray color is a bit light, at least on my monitor. :)

Reply:
Fixed!